frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -355,9 +363,9 @@ private void createClearingRequest(ResourceRequest request, ResourceResponse res
             JsonGenerator jsonGenerator = JSON_FACTORY.createGenerator(response.getWriter());
             jsonGenerator.writeStartObject();
             jsonGenerator.writeStringField(RESULT, requestSummary.getRequestStatus().toString());
-            if (AddDocumentRequestStatus.FAILURE.equals(requestSummary.getRequestStatus())) {
+            if (!AddDocumentRequestStatus.SUCCESS.equals(requestSummary.getRequestStatus())) {
                 ResourceBundle resourceBundle = ResourceBundleUtil.getBundle("content.Language", request.getLocale(), getClass());
-                jsonGenerator.writeStringField("message", LanguageUtil.get(resourceBundle, requestSummary.getMessage().replace(' ','.').toLowerCase()));
+                jsonGenerator.writeStringField(SW360Constants.MESSAGE, LanguageUtil.get(resourceBundle, requestSummary.getMessage().replace(' ', '.').toLowerCase()));
```

Comment:
Indentation, after the comma in the replace method.

Reply:
fixed the indentation.